services/event_service_test.go

```diff
@@ -1483,16 +1500,18 @@ func TestEventService_RetryEventDelivery(t *testing.T) {
 				g: &datastore.Group{UID: "abc"},
 			},
 			wantErr:    true,
-			wantErrMsg: "endpoint is being re-activated",
+			wantErrMsg: "subscription is being re-activated",
 		},
 		{
 			name: "should_retry_event_delivery_with_inactive_endpoint",
```

Comment:
Should this change to `should_retry_event_delivery_with_inactive_subscription`? as with other test cases here.